RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.23.0 (2018-01-04)
+==========================
+
+Language
+--------
+- [Fixed displaying duplicate errors on some trait bounds.][45772]
```

Comment:
https://github.com/rust-lang/rust/pull/45772 *is* a language change - arbitrary auto traits are permitted in trait objects, but not displaying duplicate errors is not a language change.